vhpc_toolkit/command.py

```diff
@@ -59,3 +59,9 @@ def command():
         ops.cluster()
     elif ops.cfg[CMD_KEY] == "get_vm_config":
         ops.get_vm_config_cli()
+    elif ops.cfg[CMD_KEY] == "secure_boot":
+        ops.secure_boot_cli()
+    elif ops.cfg[CMD_KEY] == "vm_scheduling_affinity":
```

Comment:
Let's shorten this command to sched_affinity. The current one is too long and mess up with the help msg:

get_vm_config       View the performance metrics of the VM
    view                View the vCenter object names
    clone               Clone VM(s) via Full Clone or Linked Clone
    destroy             Destroy VM(s)
    power               Power on/off VM(s)
    secure_boot         Secure boot on/off VM(s)
    vm_scheduling_affinity
                        Change VM scheduling affinity
    numa_affinity       Change NUMA node affinity
    cpumem              Reconfigure CPU/memory for VM(s)